### PR TITLE
Lots of stuff

### PR DESCRIPTION
**Pull request by schmamps**
## index.html
- fix missing </h1> tag
- add more scripts
- add more styles
- add placeholder images
## map.css
- styling for the map/hint area (different name, perhaps?)
## map.js
- loads map onto page
- plots clicked point on map
- fires callback event when point clicked
## site.css
- basic styling
## site.js
- bootstraps functionality in map.js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,16 +3,40 @@
...
+      >
+    </section>
   </main>
   <footer></footer>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
   <script src="http://cdn.jsdelivr.net/lodash/4.11.2/lodash.js"></script>
   <script src="http://openlayers.org/en/v3.17.1/build/ol.js"></script>
+  <!--script src="https://cdnjs.cloudflare.com/ajax/libs/ol3/3.17.1/ol-debug.js"></script-->
+  <script src="map.js"></script>
   <script src="site.js"></script>
```
Comment (stephenmuller) [inline index.html:41]: I put all my code in scoring.js, at some point we'll have to add that here

Comment (schmamps) [inline index.html:41]: I'll add the line to my branch and we'll merge it.

File: map.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.game {
+  display: flex;
+  height: 33%;
+  margin-top:-2em;
```
Comment (stephenmuller) [inline map.css]: What's an em?
There are a handful of non-spaced  arguments, is that intentional?

Comment (schmamps) [inline map.css]: It's a measurement relative to the size of type, which is more predictable than pixels (unless you're using actual pixels, like in images).

The (non-)spaces? Just sloppiness.

**Pull request merged**